public/director-upload-movie.php

```diff
@@ -32,26 +107,23 @@
               <span class="down-arrow">&blacktriangledown;</span>
               </div>
               <div class="list">
-                <label for="Action" class="task"><input type="checkbox" name="Category[]" id="Action" value="Action"/> Action</label>
-                <label for="Comedy" class="task"><input type="checkbox" name="Category[]" id="Comedy" value="Comedy"/> Comedy</label>
-                <label for="Drama" class="task"><input type="checkbox" name="Category[]" id="Drama" value="Drama"/> Drama</label>
-                <label for="Documentary" class="task"><input type="checkbox" name="Category[]" id="Documentary" value="Documentary"/> Documentary</label>
-                <label for="Fantasy" class="task"><input type="checkbox" name="Category[]" id="Fantasy" value="Fantasy"/> Fantasy</label>
-                <label for="Horror" class="task"><input type="checkbox" name="Category[]" id="Horror" value="Horror"/> Horror</label>
-                <label for="Mystery" class="task"><input type="checkbox" name="Category[]" id="Mystery" value="Mystery"/> Mystery</label>
-                <label for="Romance" class="task"><input type="checkbox"  name="Category[]" id="Romance" value="Romance"/> Romance</label>
-                <label for="Science-fiction" class="task"><input type="checkbox"  name="Category[]" id="Science-fiction" value="Science-fiction"/> Science fiction</label>
-                <label for="Thriller" class="task"><input type="checkbox" name="Category[]" id="Thriller" value="Thriller"/> Thriller</label>
-                <label for="Western" class="task"><input type="checkbox" name="Category[]" id="Western" value="Western"/> Western</label>
-                <label for="Other" class="task"><input type="checkbox" name="Category[]" id="Other" value="Other"/> Other</label>
+              <?php
+                $genres = getTableRecords("SELECT id, naam FROM genre");
+                foreach($genres as $key => $genre){
+                  echo "<label for=".$genre['naam']." class=\"task\"><input type=\"checkbox\" name=\"Category[]\" id=". $genre['naam']." value='". $genre['id'] ."'/> ".$genre['naam']."</label>";
+                }
+
+
+
+              ?>
               </div>
             </div>
           </div>
           <div class="spaceupload">
             <label for="AgeRating" class="spacetextupload">Age rating</label>
             <select name="AgeRating" id="AgeRating" class="styleselect">
               <option value="" disabled selected> Select your age </option>
-              <option value="0">ALL</option>
+              <option value="All">ALL</option>
```

Comment:
Dit zou wel even een getal moeten hebben als value, dus 0. Anders gaat dit niet goed in de database